0003-siren-prototype-accept-decline.adoc

```diff
@@ -0,0 +1,88 @@
+= Siren Prototype for comit-rs HTTP API - Accept/Decline actions
+
+* Authors: Franck Royer
+* Date: 2019-05-02
+
+Tracking issue: [comit-rs#917](https://github.com/comit-network/comit-rs/issues/917)
+
+== Context
+
+In order to move forward easily with Bobtimus prime we should define the siren json document of how a IN_PROGRESS response look like in each state of accept and decline, i.e. when you call GET /swaps/ for Bob.
+
+Siren repository: https://github.com/kevinswiber/siren/
+
+The prototype must conform to [Siren JSON schema](https://github.com/kevinswiber/siren/blob/master/siren.schema.json)
+
+== Research
+
+.The prototype
+[source,json]
+----
+{
+  "class": ["swaps"],
+  "entities": [
+    {
+      "class": ["swap"],
+      "rel": [],
```

Comment:
🤔 I think this is unneeded in our context but I am not sure I got the correct grasp:

> Defines the relationship of the sub-entity to its parent, per Web Linking (RFC5899).

Reply:
Can just be `item` in our case.
See here: https://github.com/kevinswiber/siren/issues/17#issuecomment-26816414

Reply:
> Did you validate that it conforms to the schema? :)

Yes I configured CLion for that.